src/txn/txn_rollback_to_stable.c

```diff
@@ -791,22 +863,24 @@ __rollback_abort_col_var(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t r
  *     Abort updates on a fixed length col leaf page with timestamps newer than the rollback
  *     timestamp.
  */
-static void
+static int
 __rollback_abort_col_fix(WT_SESSION_IMPL *session, WT_PAGE *page, wt_timestamp_t rollback_timestamp)
 {
     WT_INSERT_HEAD *ins;
 
     /* Review the changes to the original on-page data items */
     if ((ins = WT_COL_UPDATE_SINGLE(page)) != NULL)
-        __rollback_abort_insert_list(session, ins, rollback_timestamp);
+        WT_RET(__rollback_abort_insert_list(session, page, ins, rollback_timestamp, NULL));
```

Comment:
For fixed-length column store type, there is no history store used. Do we need to either don't pass the page here or add a check in the location to not allocate memory and etc?

Reply:
We only allocate an u64 integer so I think it is fine. We pay another price if we do the type check.

Reply:
ok, undertood.